sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -192,6 +199,25 @@ def __init__(self, **kwargs: Any) -> None:  # pylint: disable=too-many-statement
                 )
             else:
                 credentials.append(InteractiveBrowserCredential(tenant_id=interactive_browser_tenant_id, **kwargs))
+        broker_credential_class = _get_broker_credential()
+        if broker_credential_class and sys.platform.startswith("win"):
+            # The silent auth flow for brokered auth is only available on Windows.
+            broker_credential_args = {
+                "tenant_id": interactive_browser_tenant_id,
+                "parent_window_handle": msal.PublicClientApplication.CONSOLE_WINDOW_HANDLE,
+                "use_default_broker_account": True,
+                **kwargs,
+            }
+            if interactive_browser_client_id:
+                broker_credential_args["client_id"] = interactive_browser_client_id
+
+            credentials.append(broker_credential_class(**broker_credential_args))
+        else:
+            _LOGGER.info(
+                "InteractiveBrowserBrokerCredential unavailable. Brokered authentication in DefaultAzureCredential is "
```

Comment:
Brokered authentication is not accurate.

cli may support Brokered authentication too which is available even the identity-broker package is not installed.